### DISCUSSION TITLE
Support for old Aiptek tablets?

### DISCUSSION THREAD

**Discussion by jimbolaya** [OpenTabletDriver/OpenTabletDriver#3392] [General]
### Description

There is already a driver for the Aiptek tablets that works pretty well, but there are no user configuration tools. I am also concerned about support moving forward. I was hoping that if it was folded into an active project like this, that the OpenTablet user configuration tools could be used with this tablet.

I am happy to help do any testing, but I'm not very capable at coding. I don't know how difficult it would be to pull in the existing code to make it work with this project.

### Acknowledgements

- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [ ] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
Provided you have one of these open an issue using the configurations template.
